.circleci/config.yml

```diff
@@ -174,24 +175,65 @@ commands:
       py-version:
         <<: *py-version-template
     steps:
+      - run:
+          name: Preparing environment
+          command: |
+            export OS_NAME=$(uname -s | sed s/Darwin/macos/ | tr '[:lower:]' '[:upper:]')
+            echo "export OS_NAME=$OS_NAME" >> ${BASH_ENV}
+            if [[ ${OS_NAME} == "MACOS" ]]; then
+              echo "export PYENV_ROOT=~/.local-MACOS/.pyenv" >> ${BASH_ENV}
+              if [[ $PYTHON_VERSION_SHORT == 3.8 ]]; then
+                export PYENV_PYTHON_VERSION=$PYENV_PYTHON_VERSION_38
+              elif [[ $PYTHON_VERSION_SHORT == 3.7 ]]; then
+                export PYENV_PYTHON_VERSION=$PYENV_PYTHON_VERSION_37
+              else
+                exit 1
+              fi
+              echo "export PYENV_PYTHON_VERSION=$PYENV_PYTHON_VERSION" >> ${BASH_ENV}
+            fi
+            echo "SHA: ${CIRCLE_SHA1}"
```

Comment:
Are those good for anything or is this a leftover from testing?

Reply:
They ended up there from replicating the "run" command in `prepare-system`, since those reference the job commit and tag, you're right, they don't have to be in there twice.